types/ioredis/index.d.ts

```diff
@@ -229,7 +229,7 @@ declare namespace IORedis {
         smembers(key: KeyType, callback: (err: Error, res: any) => void): void;
         smembers(key: KeyType): Promise<any>;
 
-        zadd(key: KeyType, ...args: string[]): Promise<number | string>;
+        zadd(key: KeyType, ...args: any[]): Promise<number | string>;
```

Comment:
The CI fails.
And `(string | number)[]` seems more precise according to the document: https://redis.io/commands/zadd